src/app/login/login.component.html

```diff
@@ -0,0 +1,118 @@
+<div class="auth-page-wrapper pt-5">
+  <div class="image position">
+    <div class="bg-image"></div>
+    <div class="shape">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        version="1.1"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        viewBox="0 0 1440 120"
+      >
+        <path
+          d="M 0,36 C 144,53.6 432,123.2 720,124 C 1008,124.8 1296,56.8 1440,40L1440 140L0 140z"
+        ></path>
+      </svg>
+    </div>
+  </div>
+  <div class="auth-content">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <div class="text-center mt-sm-5 mb-4 text-white-50">
+            <div class="logo">
+              <a class="d-inline-block auth-logo" href="/">
+                <img src="../../assets/morden-logo.png" alt="" height="20" />
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-12 text-center">
+          <div class="login-card">
+            <h4>Welcome Back !</h4>
+            <form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+              <div class="form-group">
+                <label for="email">Email</label>
```

Comment:
do not pass direct strings for label and validation message pass that from ts file.
-->for all label and validation message.